.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
we should probably add `.idea` to the top level `.gitignore` and not check in any of the files in this directory